### DISCUSSION TITLE
[Slack] Invite link

### DISCUSSION THREAD

**Discussion by Shadowpii** [meshery/meshery#8108] [General] [answered]
Slack link on [https://meshery.io/](website) is expired

**Comment by akshayamadhuri** [comment]
@Shadowpii i think you can join with this  https://layer5io.slack.com/join/shared_invite/zt-1ysu95kyq-H9nDpUp71H7jUMaCnmKjAg#/shared-invite/email

**Reply by shreverr** [reply]
@akshayamadhuri this link is not working. Can you provide another link

**Reply by Jougan-0** [reply]
@shreverr 
https://join.slack.com/t/layer5io/shared_invite/zt-2kggko46a-TXQvJTN6SPVYeppZve~k6g

**Reply by shreverr** [reply]
Thanks @Jougan-0

**Reply by leecalcote** [reply, marked as answer]
Don’t share links like this. They will only expire.

https://slack.meshery.io is  the proper link. This is littered throughout every repo and all of our websites.

**Reply by vishalvivekm** [reply]
Thanks @Jougan-0.
@akshayamadhuri @shreverr and all future visitors, 

These custom invite links expire and need to be updated every 30 days. Therefore, you should always go to https://slack.meshery.io/, where we take care of updating them.
